app/controllers/application_controller.rb

```diff
@@ -74,7 +74,7 @@ def prepare_menu
   end
 
   def default_url_options(options = {})
-    { locale: I18n.locale }.merge options
+    { locale: nil }.merge options
```

Comment:
This is useless now, isn't it?
If you set the locale to nil, it's the same as not setting it at all. Merging it with options just returns options

Reply:
Sure. I didn't know how relevant this, in our case is.